pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -1492,7 +1486,10 @@ func isPodReady(pod corev1.Pod) bool {
 }
 
 func (r *ReconcileInfinispan) serviceForInfinispan(m *infinispanv1.Infinispan) *corev1.Service {
-	ls := labelsForInfinispan(m.ObjectMeta.Name)
+	lsPodSelector := labelsForInfinispan(m.ObjectMeta.Name)
```

Comment:
@rigazilla what do you think about change labelsForInfinispan func to add advanced parameters for adding flexible object label inside it like labelsForInfinispan(name, type) or add some logic inside to reduce number of code lines.

Reply:
yeah ok, done